__tests__/app.test.js

```diff
@@ -44,7 +44,8 @@ describe("app", () => {
         author: "butter_bridge",
         body: "I find this existence challenging",
         created_at: "2020-07-09T20:11:00.000Z",
-        votes: 100
+        votes: 100,
+        comment_count: "11"
```

Comment:
please don't amend previous tests when adding new features! you a new functionality should be in new tests, ensuring the old ones remain passing

Reply:
also check if it works for article_id 2